Clinical-Genomics/scout
# Partial causatives

This PR adds a functionality (fix #291)

**How to test**:
1. Install the branch on stage
1. Mark one or more (SNV) variants as causative or partial causative (SV variants fix ongoing)
1. Make sure that you have to associate a phenotype (HPO terms or OMIM diagnosis) to each partial causative.
1. Make sure case is not marked as solved when a partial causative variant is assigned.
1. Make sure that specific links to phenotypes on case page work.
1. Make sure events are correctly recorded.
1. Make sure you can remove causative and partial causative variants.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**

In the end I didn’t add the requirement of having a phenotype before marking a variant as causative, it’s already quite complicated as it is. We can take it on another PR
